observability/logging/cluster-logging-support.adoc

```diff
@@ -0,0 +1,73 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="cluster-logging-support"]
+include::_attributes/common-attributes.adoc[]
+= Support
+:context: cluster-logging-support
+
+toc::[]
+
+include::snippets/logging-supported-config-snip.adoc[]
+include::snippets/logging-compatibility-snip.adoc[]
+include::snippets/log6x-loki-statement-snip.adoc[]
+
+{logging-uc} {for} is an opinionated collector and normalizer of application, infrastructure, and audit logs. You can use it to forward logs to various supported systems.
+
+{logging-uc} is not:
+
+* A high scale log collection system
+* Security Information and Event Monitoring (SIEM) compliant
+* A "bring your own" (BYO) log collector configuration
+* Historical or long term log retention or storage
+* A guaranteed log sink
+* Secure storage - audit logs are not stored by default
+
+[id="cluster-logging-support-CRDs_{context}"]
+== Supported API custom resource definitions
+
+The following table describes the supported {logging-uc} APIs.
+
+.Loki API support states
+[cols="3",options="header"]
+|===
+|CustomResourceDefinition (CRD)
+|ApiVersion
+|Support state
+
+|`LokiStack`
+|`lokistack.loki.grafana.com/v1`
+|Supported from 5.5
+
+|`RulerConfig`
+|`rulerconfig.loki.grafana/v1`
+|Supported from 5.7
+
+|`AlertingRule`
+|`alertingrule.loki.grafana/v1`
+|Supported from 5.7
+
+|`RecordingRule`
+|`recordingrule.loki.grafana/v1`
+|Supported from 5.7
+
+|`LogFileMetricExporter`
+|`LogFileMetricExporter.logging.openshift.io/v1alpha1`
+|Supported from 5.8
+
+|`ClusterLogForwarder`
+|`clusterlogforwarder.logging.openshift.io/v1`
+|Supported from 4.5.
+|===
+
+include::modules/cluster-logging-maintenance-support-list.adoc[leveloffset=+1]
+include::modules/unmanaged-operators.adoc[leveloffset=+1]
+
+[id="cluster-logging-support-must-gather_{context}"]
+== Collecting logging data for Red Hat Support
+
+When opening a support case, it is helpful to provide debugging information about your cluster to Red{nbsp}Hat Support.
+
+You can use the xref:../../../support/gathering-cluster-data.adoc#gathering-cluster-data[must-gather tool] to collect diagnostic information for project-level resources, cluster-level resources, and each of the {logging} components.
```

Comment:
I think this xref has one too many `../` in the filepath. If I'm not mistaken, the first `../` points out of logging and into the observability directory, and the next one takes it out of observability and points to root.

```suggestion
You can use the xref:../../support/gathering-cluster-data.adoc#gathering-cluster-data[must-gather tool] to collect diagnostic information for project-level resources, cluster-level resources, and each of the {logging} components.
```

Reply:
astute observation! This would definitely have caused build errors when we merge loggin-6,.2 to main. Fixed it.